Pcap++/header/PcapFilter.h

```diff
@@ -84,6 +88,52 @@ namespace pcpp
 		virtual ~GeneralFilter();
 	};
 
+	//Used in FilterTester
+	class RawPacket;
+
+	/**
+	 * @class FilterTester
+	 * This class can be loaded with a filter string and then can be used to verify the string is valid and check if a packet matches it<BR>
+	 */
+	class FilterTester : public GeneralFilter
+	{
+#if __cplusplus > 199711L || _MSC_VER >= 1800 //Maybe this can be 1600 for VS2010
+		typedef std::unique_ptr<bpf_program> Ptr_t;
+#else
+		typedef std::auto_ptr<bpf_program> Ptr_t;
+#endif
+
+	private:
+		bool verified, verifiedResult;

```

Comment:
please follow the coding convention of private members' names starting with `m_` followed by a capital letter

Reply:
Easy fix, will do.